src/main/java/com/gmail/kutilandrej/Main.java

```diff
@@ -0,0 +1,18 @@
+package com.gmail.kutilandrej;
+
+import com.gmail.kutilandrej.lib.Injector;
+import com.gmail.kutilandrej.model.Movie;
+import com.gmail.kutilandrej.service.MovieService;
+
+public class Main {
+    private static Injector injector = Injector.getInstance("com.gmail.kutilandrej");
```

Comment:
можливо краще ще додати cinema, що скажеш?
```suggestion
    private static Injector injector = Injector.getInstance("com.gmail.kutilandrej.cinema");
```

Reply:
Гараазд, тут доречно мати назву проекту в імені проекту.